res/openvpn-gui-res-no.rc

```diff
@@ -344,24 +343,24 @@ BEGIN
     IDS_ERR_OLD_PWD_INCORRECT "Du har oppgitt feil passord."
     IDS_ERR_OPEN_WRITE_KEY "En feil oppsto under åpning av nøkkel-fil for skrivning (%s)."
     IDS_ERR_WRITE_NEW_KEY "En feil oppsto under opprettelse av ny nøkkel-fil (%s)."
-    IDS_NFO_PWD_CHANGED "Dn passord har blitt endret."
+    IDS_NFO_PWD_CHANGED "Passordet er blitt endret."
     IDS_ERR_READ_PKCS12 "Feil under lesning fra pkcs #12 fil (%s)."
     IDS_ERR_CREATE_PKCS12 "En feil oppsto under opprettelse av pkcs12 objekt."
-    IDS_ERR_OPEN_CONFIG "En feil oppsto under åpning av følgende konfig-fil: %s."
-    IDS_ERR_ONLY_ONE_KEY_OPTION "Du kan ikke ha mer enn en ""key"" parameter i din konfig-fil."
+    IDS_ERR_OPEN_CONFIG "En feil oppsto under åpning av følgende konfigurasjonsfil: %s."
+    IDS_ERR_ONLY_ONE_KEY_OPTION "Du kan ikke ha mer enn én ""key"" parameter i din konfig-fil."
     IDS_ERR_ONLY_KEY_OR_PKCS12 "Du kan ikke ha både ""key"" og ""pkcs12"" parametetre i din konfig-fil."
     IDS_ERR_ONLY_ONE_PKCS12_OPTION "Du kan ikke ha mer enn en ""pkcs12"" parameter i din konfig-fil."
     IDS_ERR_HAVE_KEY_OR_PKCS12 "Din konfig-fil mangler ""key"" eller ""pkcs12"" parameter."
-    IDS_ERR_KEY_FILENAME_TO_LONG "Den oppgitte path til din nøkkel i konfig-fil er for lang!"
-    IDS_ERR_PASSPHRASE2STDIN "En feil oppsto når passordet skulle sendes til OpenVPN via stdin."
-    IDS_ERR_AUTH_USERNAME2STDIN "En feil oppsto når brukernavnet skulle sendes til OpenVPN via stdin."
-    IDS_ERR_AUTH_PASSWORD2STDIN "En feil oppsto når passordet skulle sendes til OpenVPN via stdin."
-    IDS_ERR_CR2STDIN "En feil oppsto når CR skulle sendes til OpenVPN via stdin."
+    IDS_ERR_KEY_FILENAME_TO_LONG "Den oppgitte filstien til din nøkkel i konfigurasjonsfilen er for lang!"
```

Comment:
Should be more precise with 'nøkkel'.  I'd recommend 'krypteringsnøkkel'.
@selvanair Is IDS_ERR_KEY_FILENAME_TO_LONG used for anything else than just key files?  Or also CA/cert files?